Kudu.Contracts/SiteExtensions/SiteExtensionInfo.cs

```diff
@@ -40,6 +40,7 @@ public SiteExtensionInfo(SiteExtensionInfo info)
             LocalIsLatestVersion = info.LocalIsLatestVersion;
             LocalPath = info.LocalPath;
             InstalledDateTime = info.InstalledDateTime;
+            InstallationArgs = info.InstallationArgs;
```

Comment:
So @william-xie how the scenario of "adding features" will look like using ARM? If customer wants to enable an extra feature for Application Insights extension - will the script need to uninstall the extension and then install with the updated set of arguments?